### PR TITLE
AF-1130: Remove from create assets list Project, Case Project and Import Asset

### PR DESCRIPTION
**Pull request by paulovmr**
Part of an ensemble:
* https://github.com/kiegroup/kie-wb-common/pull/1644
* https://github.com/kiegroup/jbpm-wb/pull/1073

### CODE REVIEW

**Comment by paulovmr** [comment]
jenkins execute full downstream build

**Comment by paulovmr** [comment]
Hi @manstis , actually the upload was only removed from the Add Asset screen, but it can still be accessed in the project screen options.

image

**Comment by manstis** [comment]
Thanks for the explanation and clarification @paulovmr :-)

Stepping down to DEFCON 5.

**Comment by tarilabs** [comment]
> the upload was only removed from the Add Asset screen, but it can still be accessed in the project screen options (as "Import Asset").

phew!
image

**Comment by paulovmr** [comment]
jenkins execute full downstream build

**Comment by paulovmr** [comment]
jenkins execute full downstream build

**Comment by paulovmr** [comment]
jenkins execute full downstream build

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/screens/assets/add/AddAssetScreen.java [hunk 1]
```diff
@@ -47,6 +48,8 @@
         owningPerspective = LibraryPerspective.class)
 public class AddAssetScreen {
 
+    public static List<String> BLACKLISTED_RESOURCES = Arrays.asList("NewWorkspaceProjectHandler", "NewFileUploader", "NewCaseProjectHandler");
```
Comment (ederign) [inline kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/screens/assets/add/AddAssetScreen.java]: Hi Paulo, I would implement this differently due to the lack of encapsulation and also weak typing:

ResourceHandler with isProjectAsset default true and false on this 3 assets.

What do you think?

Comment (paulovmr) [reply kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/screens/assets/add/AddAssetScreen.java]: I agree, will make the refactor.

Comment (paulovmr) [reply kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/screens/assets/add/AddAssetScreen.java]: Done.

**Comment by paulovmr** [comment]
jenkins execute full downstream build

**Comment by ederign** [comment]
@barboras7 could you please take a look at this?

**Comment by barboras7** [comment]
jenkins execute full downstream build

**Pull request merged**